algorithms/schema-free/GSQL_jaccard_similarity_batch.gsql

```diff
@@ -0,0 +1,88 @@
+CREATE QUERY GSQL_jaccard_similarity_batch (
```

Comment:
@wyatt-joyner-tg I believe the changes can applied to both current Jaccard and Cosine Similarities. Since it has been proved that it is better in every way. Let's replace the current queries with your new query.